Sources/Fluent/Schema/Schema+Creator.swift

```diff
@@ -15,13 +15,14 @@ extension Schema {
         
         public func id(
```

Comment:
Right, I think that makes sense. Could take it even further and instead of overloading, only allow passing in an entity type?